core/src/oracle_config.rs

```diff
@@ -125,6 +127,10 @@ lazy_static! {
     pub static ref ORACLE_CONFIG: OracleConfig = OracleConfig::load().unwrap();
     pub static ref MAYBE_ORACLE_CONFIG: Result<OracleConfig, String> =
         OracleConfig::load().map_err(|e| e.to_string());
+    pub static ref SAFE_USER_MIN: BoxValue = MAYBE_ORACLE_CONFIG
```

Comment:
I suggest to name it `BASE_FEE` (close to `OracleConfig::base_fee`). `SAFE_USER_MIN` name comes from a box value minimal value limit (to prevent dust) and has nothing to do with tx fee. 
The use of `BoxValue::SAFE_USER_MIN` for tx fee throughout the code was incorrect in the first place and should be avoided. There is `ergo_lib::wallet::tx_builder::SUGGESTED_TX_FEE()` intended for this case. Please, use it if config is unavailable.

Reply:
You are right. Changes have been made.